green_bit_llm/evaluation/evaluate.py

```diff
@@ -21,16 +21,20 @@
 from pathlib import Path
  
 from lm_eval import evaluator
-
+from vllm.model_executor.layers.logits_processor import _apply_logits_processors
```

Comment:
I assume there is no "safe" alternativ instead of using this internal method, right? If so, I think we can use it.